Module.php

```diff
@@ -144,8 +144,9 @@ public function bootstrap($app)
         $app->on(Application::EVENT_BEFORE_REQUEST, function () use ($app) {
             $app->getView()->on(View::EVENT_END_BODY, [$this, 'renderToolbar']);
         });
-
-        $app->getUrlManager()->addRules([
+        $urlManager = $app->getUrlManager();
+        $urlManager->ruleConfig['class'] = 'yii\web\UrlRule';
```

Comment:
@lichunqiang the thing is that `bootstrap()` happends regardless of the fact that module is currnently running. That means it would affect other app parts which isn't desired.

I propose specifying rules as arrays defining class explicitly:
http://www.yiiframework.com/doc-2.0/yii-web-urlrule.html